front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -64,9 +80,25 @@
     padding-bottom: 0.5rem;
     display: flex;
     flex-direction: column;
+    &.scenario-sidemenu-v2 {
+      display: flex;
+      flex-direction: column;
```

Comment:
I think these two lines can now be dropped, because v1 has the same rules too.

Reply:
Done